docs/docs-beta/docs/tutorials/category-two/llm-fine-tuning/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: LLM Fine Tuning
+description: Learn how to fine tune an LLM
+last_update:
+   author: Dennis Hume
+sidebar_position: 10
+sidebar_class_name: hidden
+---
+
+# Fine-Tune an LLM
+
+In this tutorial, you'll build an pipeline with Dagster that:
+
+- Imports data from Goodreads DuckDB
+- Does feature engineering to enhance the data
+- Creates and validates a training and validation file
```

Comment:
@dehume I'll likely know what this means once I go through the code, but at this point I'm not entirely sure what this means.